helpers/mathjax.js

```diff
@@ -158,6 +183,23 @@ export function loadMathJax(mathJaxConfig) {
 		}
 	};
 
+	if (mathJaxConfig && mathJaxConfig.deferTypeset && !document.head.querySelector('#d2l-mathjax-fonts') && !document.head.querySelector('#MJX-CHTML-styles')) {
```

Comment:
`MJX-CHTML-styles` is the `id` of the stylesheet that MathJax applies, which includes the font at-rules. If it exists in the document head already, we don't need to add this new style. Notably, we also don't want to use this `id` because MathJax does a lot more than just the at-rules with those styles, so we also don't want to clobber MathJax's styles with these.